R/liger.R

```diff
@@ -1702,21 +1698,21 @@ runTSNE <- function(object, use.raw = F, dims.use = 1:ncol(object@H.norm), use.p
   }
   if (method == 'Rtsne') {
     set.seed(rand.seed)
-    object@tsne.coords <- Rtsne(data.use[, dims.use], pca = use.pca, check_duplicates = F, 
-                                theta = theta, perplexity = perplexity)$Y
+    object@dr.coords["tsne"] <- Rtsne(data.use[, dims.use], pca = use.pca, check_duplicates = F, 
```

Comment:
Be careful about how we're accessing list elements -- in general, we should only index with double brackets `[[]]`, as this allows us to return the actual object contained within the list. Using single brackets returns a list with a single element (the thing you're trying to access), and unfortunately this will break several of our functions which are expecting objects of another class. [Here](https://stackoverflow.com/questions/1169456/the-difference-between-bracket-and-double-bracket-for-accessing-the-el)'s a more detailed explanation in case this wasn't clear. 
Make sure to replace all instances of this kind of list accession with double brackets!